frontend/packages/ceph-storage-plugin/src/components/ocs-install/attached-devices/create-sc/wizard-pages/donut-chart.tsx

```diff
@@ -56,6 +59,12 @@ export const DiscoveryDonutChart: React.FC<DiscoveryDonutChartProps> = ({ state,
     dispatch,
   ]);
 
+  React.useEffect(() => {
+    const filteredNodes = new Set<string>();
+    state.filteredDiscoveries.forEach((discovery) => filteredNodes.add(discovery.node));
+    dispatch({ type: 'setFilteredNodes', value: Array.from(filteredNodes) });
```

Comment:
Array -> Set -> Array...? _.uniqBy

Reply:
Its just `Set -> Array` and its fine to use this way.